tests/test_check.py

```diff
@@ -1,13 +1,25 @@
 import os
 import shutil
 import unittest
-from base import *
+
+from tests.base import (
+    HPSS_ARCHIVE,
+    TOP_LEVEL,
+    ZSTASH_PATH,
+    TestZstash,
+    compare,
+    print_in_box,
+    print_starred,
+    run_cmd,
+    write_file,
+)
```

Comment:
from module import * is discouraged because the programmer often won’t know where an imported object is defined. The imported module, class, or function should be explicitly defined.
https://www.flake8rules.com/rules/F403.html

In our case, explicit imports fix flake8 and mypy issues, and the various test files often only use a subset of the imports.